spec/javascripts/fixtures/json/listings-api-show.json

```diff
@@ -1 +1,253 @@
\ No newline at end of file
+{
+  "listing": {
+    "unitSummaries": {
+      "reserved": null,
+      "general": [
+        {
+          "unitType": "1 BR",
+          "totalUnits": 1,
+          "priorities": null,
+          "minSquareFt": null,
+          "minRentalMinIncome": 5255.0,
+          "minPercentIncome": null,
+          "minOccupancy": 1,
+          "minMonthlyRent": 2102.0,
+          "maxSquareFt": null,
+          "maxRentalMinIncome": 5255.0,
+          "maxPercentIncome": null,
+          "maxOccupancy": 3,
+          "maxMonthlyRent": 2102.0,
+          "listingID": "a0W0P00000F8YG4UAN"
+        }
+      ]
+    },
+    "reservedDescriptor": null,
+    "prioritiesDescriptor": null,
+    "listingID": "a0W0P00000F8YG4UAN",
+    "doesMatch": null,
+    "chartTypes": [
+      {
+        "year": 2016,
+        "percent": 100,
+        "numOfHousehold": null,
+        "chartType": "HUD Unadjusted",
+        "amount": null
+      }
+    ],
+    "attributes": {
+      "type": "Listing",
+      "url": "/services/data/v35.0/sobjects/Listing/a0W0P00000F8YG4UAN"
+    },
+    "Name": "Automated Test Listing (please do not modify)",
+    "First_Come_First_Served": false,
+    "Building": {
+      "attributes": {
+        "type": "Building",
+        "url": "/services/data/v35.0/sobjects/Building/a0a0P00000GbJmbQAF"
+      },
+      "Id": "a0a0P00000GbJmbQAF"
+    },
+    "Tenure": "New rental",
```

Comment:
this was the only change I made to this file other than pretty-printing it so it's easier to read and see changes in the future.